glide.yaml

```diff
@@ -10,7 +10,7 @@ excludeDirs:
 - toolbox
 import:
 - package: github.com/coreos/etcd
-  version: release-3.1
+  version: 20490caaf0dcd96bb4a95e40625559def8ef5b04
```

Comment:
why did etcd need to be updated? or is this hash equivalent to `release-3.1` tag?  note we use etcd internally for standalone, so if it did actually change it would be good to test standalone for sanity.